packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -28,15 +29,29 @@ const double _kArrowScreenPadding = 26.0;
 // Values extracted from https://developer.apple.com/design/resources/.
 const Radius _kToolbarBorderRadius = Radius.circular(8);
 
+// Color was measured from a screenshot of iOS 16.0.2
+// TODO(LongCatIsLooong): https://github.com/flutter/flutter/issues/41507.
+const CupertinoDynamicColor _kToolbarBackgroundColor =
+    CupertinoDynamicColor.withBrightness(
+  color: Color(0xFFF6F6F6),
+  darkColor: Color(0xFF222222),
+);
```

Comment:
Nit: I would indent these 3 lines by 4 spaces.

Reply:
I could format this as the other `CupertinoDynamicColor`s are formatted too, like this:

```dart
const CupertinoDynamicColor _kToolbarBackgroundColor = CupertinoDynamicColor.withBrightness(
  color: Color(0xFFF6F6F6),
  darkColor: Color(0xFF222222),
);
```

What do you think?

Reply:
I ended up doing what @tgucio suggested as it was already how the other `CupertinoDynamicColor`s were formatted.